### PR TITLE
warriors: update and activate commerce tests

### PR DESCRIPTION
**Pull request by afmicka**
CORS issues for commerce prices and CTAs have been removed. 
Updating and activating commerce tests for milo PRs.

### CODE REVIEW

**Comment by afmicka** [comment]
@Dli3 @JackySun9 @skumar09 

Today has been merged to milo main the [PR](https://github.com/adobecom/milo/pull/2725) that removes CORS issues for commerce prices and CTAs on branches. I have updated and activated commerce tests so that they could run on PRs now. When i run locally, those tests are passing.

However, here they are failing the same as before. Does this repo needs some code syncing with milo repo? Do you have idea what could cause that the prices are still not visible for the run here? 

One of the tests pages for example: https://main--milo--adobecom.hlx.live/drafts/nala/features/commerce/promo-placeholders (prices are visible now)

**Comment by afmicka** [comment]
@Dli3 @skumar09  Skipped the tests for chromium. I see merch-block is also skipped some time ago, this issue for chromium is not new.

File: tests/milo/commerce.feature.test.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ test.beforeEach(async ({ page, baseURL }) => {
 });
 
 test.describe('Commerce feature test suite', () => {
+  test.skip(({ browserName }) => browserName === 'chromium', 'Skipping tests for Chrome browser');
```
Comment (Dli3) [inline tests/milo/commerce.feature.test.js:22]: If this works locally for you, you could add a condition to skip if process.env.ci is true and it's running on chromium. This way, you can still run against chromium locally.
@afmicka

Comment (afmicka) [reply tests/milo/commerce.feature.test.js]: @Dli3 did i add the condition wrongly? It did not help, failed here again.

**Comment by JackySun9** [comment]
please fix all lint errors

File: tests/milo/commerce.feature.test.js (hunk 1 above)
Comment (Dli3) [reply tests/milo/commerce.feature.test.js]: Looks like process.env.ci didn't work here. But `if (browserName === 'chromium' && process.env.GITHUB_ACTIONS) test.skip()` did.

**Comment by afmicka** [comment]
Thanks @JackySun9 . Was trying different approaches with @Dli3 to skip tests only on github actions. We found the right one and all is good. 

Once approved, will merge it.

**Review by JackySun9 — APPROVED** [review]

**Pull request merged**